admin/collection_providers/forms.py

```diff
@@ -226,7 +227,7 @@ def clean_study_design_choices(self):
         primary_collection = collection_provider.primary_collection
         if primary_collection:  # Modifying an existing CollectionProvider
             old_choices = {c.strip(' ') for c in primary_collection.study_design_choices}
-            updated_choices = {c.strip(' ') for c in json.loads(self.data.get('study_design_choices'))}
+            updated_choices = {c.strip(' ') for c in json.loads(self.data.get('study_design_choices') or '{}')}
```

Comment:
Considering all of the other fields are being handled this same way, I *think* that the problem might actually be here: https://github.com/CenterForOpenScience/osf.io/blob/develop/admin/collection_providers/views.py#L218